packages/invoices/src/OutboundShipment/DetailView/modals/BatchesTable.tsx

```diff
@@ -1,64 +1,32 @@
 import React from 'react';
 import {
+  NumericTextInput,
   Divider,
-  FieldValues,
   isAlmostExpired,
-  ModalNumericInput,
   Table,
   TableBody,
   TableCell,
   TableCellProps,
   TableContainer,
   TableHead,
   TableRow,
-  UseFormRegister,
-  useFormContext,
   useFormatDate,
   useTranslation,
   ReadOnlyInput,
   InvoiceNodeStatus,
 } from '@openmsupply-client/common';
-import { BatchRow } from '../../../types';
-import { PackSizeController } from './ItemDetailsModal';
-
+import { DraftOutboundLine } from '../../../types';
```

Comment:
I renamed `BatchRow` -> `DraftOutboundLine` similar naming of the other pages for consistency and think it's more descriptive